internal/tenant/model.go

```diff
@@ -2,35 +2,44 @@ package tenant
 
 import "time"
 
-type TenantInfoList struct {
-	Data []TenantInfoData
+// InfoList defines list of informations about a tenant
```

Comment:
This is indeed the naming convention as per go best practices, but I'm surprised to see you rename just one of these instead of doing it everywhere. Progress indeed, so LGTM.